x-pack/plugins/ml/public/application/components/job_spaces_repair/repair_results.tsx

```diff
@@ -0,0 +1,178 @@
+
+      <DatafeedsAdded repairItems={repairItems} />
+
+      <EuiHorizontalRule margin="l" />
+
+      <DatafeedsRemoved repairItems={repairItems} />
+
+      <EuiHorizontalRule margin="l" />
+    </>
+  );
+};
+
+const SavedObjectsCreated: FC<{ repairItems: RepairSavedObjectResponse }> = ({ repairItems }) => {
+  const items = Object.keys(repairItems.savedObjectsCreated);
+
+  const title = (
+    <>
+      <EuiTitle size="xs">
+        <h3>
+          <EuiTextColor color={items.length ? 'default' : 'subdued'}>
+            <FormattedMessage
+              id="xpack.ml.management.repairSavedObjectsFlyout.savedObjectsCreated.title"
+              defaultMessage="Missing job saved objects ({count})"
+              values={{ count: items.length }}
+            />
+          </EuiTextColor>
+        </h3>
+      </EuiTitle>
+      <EuiText size="s">
+        <p>
+          <EuiTextColor color="subdued">
+            <FormattedMessage
+              id="xpack.ml.management.repairSavedObjectsFlyout.savedObjectsCreated.description"
+              defaultMessage="Jobs exist which do not have accompanying saved objects. These will be created in the current space"
+            />
+          </EuiTextColor>
+        </p>
+      </EuiText>
+    </>
+  );
+  return <RepairItem id="savedObjectsCreated" title={title} items={items} />;
+};
+
+const SavedObjectsDeleted: FC<{ repairItems: RepairSavedObjectResponse }> = ({ repairItems }) => {
+  const items = Object.keys(repairItems.savedObjectsDeleted);
+
+  const title = (
+    <>
+      <EuiTitle size="xs">
+        <h3>
+          <EuiTextColor color={items.length ? 'default' : 'subdued'}>
+            <FormattedMessage
+              id="xpack.ml.management.repairSavedObjectsFlyout.savedObjectsDeleted.title"
+              defaultMessage="Unmatched job saved objects ({count})"
+              values={{ count: items.length }}
+            />
+          </EuiTextColor>
+        </h3>
+      </EuiTitle>
+      <EuiText size="s">
+        <p>
+          <EuiTextColor color="subdued">
+            <FormattedMessage
+              id="xpack.ml.management.repairSavedObjectsFlyout.savedObjectsDeleted.description"
+              defaultMessage="Saved objects which do not have an accompanying job. These will be left over when deleting anomaly detection jobs."
+            />
+          </EuiTextColor>
+        </p>
+      </EuiText>
+    </>
+  );
+  return <RepairItem id="savedObjectsDeleted" title={title} items={items} />;
+};
+
+const DatafeedsAdded: FC<{ repairItems: RepairSavedObjectResponse }> = ({ repairItems }) => {
+  const items = Object.keys(repairItems.datafeedsAdded);
+
+  const title = (
+    <>
+      <EuiTitle size="xs">
+        <h3>
+          <EuiTextColor color={items.length ? 'default' : 'subdued'}>
+            <FormattedMessage
+              id="xpack.ml.management.repairSavedObjectsFlyout.datafeedsAdded.title"
+              defaultMessage="Job saved objects with missing datafeeds ({count})"
```

Comment:
Per earlier suggestions, I'm not a fan of "job saved objects":
```suggestion
              defaultMessage="Saved objects with missing datafeeds ({count})"
```
Also, is there a reason this needs a separate section?  Couldn't it just be grouped with the other "Missing saved objects"?

Reply:
they are different tasks, this task does not create a saved object, it adds a datafeed id to a saved object if it is missing and the datafeed exists.
It makes sense to be to keep them separate.

Reply:
updated in https://github.com/elastic/kibana/pull/83320/commits/29c6175bd0c1efa92d388f57dfb34a8fcb23e47e